cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,66 @@
+name: o3de
+version: '${O3DE_INSTALL_VERSION_STRING}'
+summary: O3DE Engine
+description: |
```

Comment:
Not sure if we need to, but would it be worth including the `icon:` file for the store here as well?

Here's the Windows one: https://github.com/o3de/o3de/blob/development/cmake/Platform/Windows/Packaging/product_logo.png

Reply:
I'll add this along with the store description in a follow up PR